alvr/client_openxr/src/lobby.rs

```diff
@@ -95,6 +99,24 @@ impl Lobby {
             vec![crate::default_view(), crate::default_view()]
         };
 
+        self.xr_session
```

Comment:
Can't the device potentially return hand data when the hand is gripping the controller (just based on position relative to the controller) or is that not an issue?

Reply:
What you are talking about is hybrid controllers and currently not supported but requested. Currently the implementation of `get_hand_motion()` always returns a global hand/controller pose if hand tracking data is present, using the palm joint.